modules/virt-template-fields-web.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+The following table outlines the fields when creating a new custom template in the {VirtProductName} web console. Not all fields are editable.
+
+[id="virt-template-fields-web_{context}"]
+= Custom template web console fields
```

Comment:
We need to use this module for both edit VMs and create VMs. I recommend a more generic title like "Virtual machine fields". Also, there is no need to mention the web console. It's clear from the title.